daemon/daemon.go

```diff
@@ -64,16 +74,27 @@ func Start(d Daemon, wait bool) error {
 		// Wait for the process to exit
 		err = cmd.Wait()
 		return err
+	} else {
+		// If the process exits at some point later, do read it's
+		// exit status. This should not let it be a zombie.
+		done := make(chan error, 1)
+		go func() {
+			done <- cmd.Wait()
```

Comment:
You don't need a channel here. You can just directly assign to `err`. As a thumb rule, you should be using channels when communicating across channels most of the time.